packages/dcos-event-stream/src/MesosOperatorApiClient.js

```diff
@@ -0,0 +1,74 @@
+import http from "http";
+import Rx from "rxjs/Rx";
+import requestTemplate from "./constants/MesosOperatorRequest";
+
+// *******************************************************
+// TODO: CHECK HOW TO USE THE CONNECTION MODULE WITH THIS
```

Comment:
You can either check if XHRConnection will fit your need (we can edit it, if needed[1]) or create a new subclass of AbstractConnection[2] and use it like this:
```
const connect = () => {
  const req = new XHRConnection(
    "/mesos/api/v1", 
    "POST", 
    JSON.stringify({type: "SUBSCRIBE"}), 
    "application/json"
  );
  req.on("progress", (req) => {
    parseResponse(req.response);
  });
  req.on("error", (req) => {
    console.error(`problem with request, received server code:`, req.status);
  });
  ConnectionManager.queue(req);
}
```
you also have `open` and `close` events, if you need to do something after opening or closing the request and you can call req.close() to kill it (but then you have to init the `const req` outside of `connect()` obviously..)

[1] we should edit XHRConnection to receive a headers-object instead of content-type
[2] please keep in mind that XHRConnection & AbstractConnection are extremely alpha at this point, so if you need them to do something specific - please comment in my PR #2361 
also they still use that this.protected context stuff, i'll change this next week